environment/src/test/java/com/sequenceiq/environment/environment/flow/deletion/handler/ResourceEncryptionDeleteHandlerTest.java

```diff
@@ -160,6 +160,17 @@ void testDeletionContinuesIfDiskEncryptionSetIdIsNotPresent() {
         verifyEnvDeleteEvent();
     }
 
+    @Test
+    void testErrorWhenDeleteEncryptionResourcesResultsInErrorWhileEnvironmentSave() {
+        IllegalArgumentException errorOnSave = new IllegalArgumentException("error");
+        Environment environment = new Environment();
+        when(environmentService.findEnvironmentById(ENVIRONMENT_ID)).thenReturn(Optional.of(environment));
+        when(environmentService.save(any(Environment.class))).thenThrow(errorOnSave);
```

Comment:
Please replace `any(Environment.class)` with `environment`. We know exactly which instance to expect in the invocation, so let us use it. This is a general advice for all unit tests.